.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "Xamarin.Auth"]
```

Comment:
Explanation:
1. Xamarin.Auth was used as submodule in earlier version (on master branch), so I just moved it to external/ folder to match our CI system
2. during development it helps to reference projects for easier debugging and fixing
3. there are still features to be added (Windows platforms) to Xamarin.Social, so it would be beneficial to keep Xamarin.Auth as a submodule for a while